src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        String[] resultList = new String[]{};
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String readerValue = reader.readLine();
+            String regex = "[?.:!;]";
+
+            if (readerValue == null) {
+                return new String[]{};
+            }
+
+            while (readerValue != null) {
+                String[] readerValueList = readerValue.split(" ");
```

Comment:
```suggestion
            String regex = "[?.:!;]";
            if (readerValue == null) {
                return new String[]{};
            }
            while (readerValue != null) {
                String[] readerValueList = readerValue.split(" ");
```
there is no need to put these empty lines, they make code unnecessary longer